### PR TITLE
Features/hull upgrade dev

### PR DESCRIPTION
**Pull request by mkawalek**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,42 +1,22 @@
-import express from "express";
-import bodyParser from "body-parser";
-import path from "path";
-import ejs from "ejs";
-
 import devMode from "./util/dev-mode";
 import SyncAgent from "./sync-agent";
-
 import KueRouter from "./util/kue-router";
+import bodyParser from "body-parser";
```
Comment (michaloo) [inline server/server.js]: @mkawalek we tend to keep external dependencies imported before internal.

So every `import from "package-name"` should be before any `import from "./local-file"`

File: server/server.js [hunk 2]
```diff
@@ -72,30 +52,20 @@ module.exports = function server(options = {}) {
...
 
-  app.post("/import", checkConfiguration, ({ agent }, res) => {
-    agent.async("startImport");
+  app.post("/import", checkConfiguration, (req, res) => {
+    req.hull.enqueue("startImport");
     res.json({ status: "scheduled" });
   });
 
-  app.post("/sync", checkConfiguration, ({ agent }, res) => {
+  app.post("/sync", checkConfiguration, ({ agent }, req, res) => {
```
Comment (michaloo) [inline server/server.js]: The signature of the handler callback is different here than for `/import` route.
Are you sure that the `/sync` works? you can trigger and test it using some rest API client (e.g. curl), but don't forget about the credentials.

File: server/sync-agent.js [hunk 3]
```diff
@@ -74,16 +38,15 @@ export default class SyncAgent {
...
     // Expose the ship settings
     // and the Hull instance.
     this.ship = ship;
     this.hull = client;
-    this.queue = queue;
     this.job = job;
     this.batchSize = batchSize;
 
-    this.importDelay = _.random(0, 120);
+    this.importDelay = _.random(0, process.env.IMPORT_DELAY || 120);
```
Comment (michaloo) [inline server/sync-agent.js]: let's add the new env var to `/app.json` file

**Comment by mkawalek** [comment]
@michaloo I fixed all indicated issues :)

**Comment by michaloo** [comment]
Since we have https://github.com/hull-ships/hull-sql/pull/10 we can close this one.

**Pull request closed without merging**